src/inverters/dto/custom-entity.ts

```diff
@@ -53,8 +53,8 @@ export function convertToCustomEntity(entity: any): CustomEntity {
         ...entity,
         toNum: (decimals?: number, invert?: boolean) => Utils.toNum(entity?.state, decimals, invert),
         isValid: () => entity?.state !== null && entity.state !== undefined && entity.state !== 'unknown' || false,
-        notEmpty: () => entity?.state !== '' || false,
-        isNaN: () => Number.isNaN(entity?.state) || true,
```

Comment:
Can you test isNaN: () => !Number.isNaN(entity?.state), This will return true if entity.state is NaN or undefined.

Reply:
yes this will definitely work correctly